enterprise/server/githubapp/githubapp.go

```diff
@@ -659,6 +759,10 @@ func (a *GitHubApp) GetAccessibleGitHubRepos(ctx context.Context, req *ghpb.GetA
 }
 
 func (a *GitHubApp) CreateRepo(ctx context.Context, req *rppb.CreateRepoRequest) (*rppb.CreateRepoResponse, error) {
+	if a.readOnly {
+		return nil, status.PermissionDeniedError("not supported for read-only app")
+	}
```

Comment:
do we need these checks or can we just let the GitHub API return a permissions error?

Reply:
I thought it might be nice to have a slightly more descriptive error indicating you installed the wrong app, as opposed to any assortment of permissions based errors